core/src/main/kotlin/org/eln2/space/Location.kt

```diff
@@ -134,6 +198,9 @@ enum class PlanarFace(val int: Int) {
 			})
 		}
 
+		/**
+		 * Gets the plane that the normal of the vector is pointing to.
```

Comment:
not quite "normal of the vector"; this is more "get the planar face with a normal closest in alignment to this vector". The face's normals point inward--toward the center--and so these tend to be the opposite of their axis _from_ the center.